pkg/cmd/deploy/requests.go

```diff
@@ -274,7 +274,7 @@ func (cmd *DeployCmd) doOriginSingle(
 	ctx context.Context,
 	conf *contracts.AzionApplicationOptions,
 	msgs *[]string) (int64, error) {
-	var DefaultOrigin = [1]string{"httpbin.org"}
```

Comment:
Farewell, old friend. 
Press F to pay respects.

![](https://media1.tenor.com/m/H5ilkYCf6iAAAAAC/anime-manga.gif)